There are several possible solutions to your problem. Here are a few things you can try:

1. **Check your Python interpreter**: Make sure that the Python interpreter you're using in VS Code is the same one you're using in your 'da38' environment. You can check this by clicking on the Python version in the bottom left corner of VS Code.

2. **Check your Jupyter server**: Try to start your Jupyter server manually from the command line using `jupyter notebook`. If this works, it means that the problem is likely with VS Code, not Jupyter.

3. **Reinstall the Python extension**: Uninstall and reinstall the Python extension in VS Code.

4. **Update your packages**: Make sure that all your packages are up to date. You can do this by running `conda update --all` in your 'da38' environment.

5. **Reinstall Jupyter**: Uninstall Jupyter using `conda uninstall jupyter` and then reinstall it using `conda install jupyter`.

6. **Check your firewall settings**: Sometimes, firewall settings can prevent VS Code from connecting to the Jupyter server. Make sure that VS Code is allowed through your firewall.

7. **Check your VS Code settings**: In your VS Code settings (`.vscode/settings.json`), make sure that the `python.pythonPath` is set to the Python interpreter of your 'da38' environment.

If none of these solutions work, it might be helpful to post your problem on the [VS Code GitHub page](https://github.com/microsoft/vscode) or on [Stack Overflow](https://stackoverflow.com/).